framework/files/usr/sbin/elemental-populate-node-labels

```diff
@@ -6,10 +6,10 @@ set -e
 IMAGE=$(echo "$IMAGE" | sed 's![/:]!-!g')
 for i in rke2 k3s; do
     mkdir -p /etc/rancher/$i/config.yaml.d
-    cat > /etc/rancher/$i/config.yaml.d/99-rancheros-labels.yaml << EOF
+    cat > /etc/rancher/$i/config.yaml.d/99-elemental-labels.yaml << EOF
 node-label+:
-- rancheros.cattle.io/image=${IMAGE}
-- rancheros.cattle.io/tag=${IMAGE_TAG}
+- elemental.cattle.io/image=${IMAGE}
+- elemental.cattle.io/tag=${IMAGE_TAG}
```

Comment:
@kkaempf do we have the `IMAGE_TAG` in the os-release generated by OBS?

This file is sourcing `rancheros-release` file at the moment.

Reply:
nope, but we can just move that to be on os-release with a small change so this should be good to go once we change that